### ISSUE TITLE
TASK: Provide way to customize package comparing to Fedora

### ISSUE THREAD

**Issue by voxik** [fedora-eln/eln#55]
### What does the ELN SIG need to do?

Recently, my request for minimizing package dependencies was denied in Fedora [1]. I completely understand the reasons. Nevertheless, I'd still love to have a way to override Fedora and I see several options:

1) Have ELN branch for such cases. This would need to be approved. The additional commits would need to be rebased on top of the Fedora changes and if the rebase fails, the appropriate maitainer would be notified.
2) Provide overrides content resolver. This way, I'd be able to remove (or add) dependencies to certain Fedora packages as needed.
3) Provide better placeholders for content resolver. The current placeholders are not enough.

[1] https://src.fedoraproject.org/rpms/rubygem-nokogiri/pull-request/5

**Comment by sgallagher** [comment]
@voxik I've opened #56 that I think will be closely related to this ticket.

**Comment by voxik** [comment]
Just FTR, this is IMO interesting case:

https://src.fedoraproject.org/rpms/rubygem-asciidoctor/pull-request/6

In such case

1. rubygem-asciidoctor is available in ELN
2. it is not shipped in RHEL, because it is only part of buildroot
3. therefore it can be added into EPEL

So here the EPEL and ELN/RHEL dependencies might need to differ.

**Comment by hroncok** [comment]
> So here the EPEL and ELN/RHEL dependencies might need to differ.

So can the conditionals. When forced to conditionalize for RHEL 9, I've used `%if 0%{?rhel} >= 9 && !0%{?epel}`

**Comment by voxik** [comment]
Right, there is certainly way to do it. But there is no way to do it while keeping sanity.
